packages/odyssey-storybook/src/components/Infobox/Infobox.stories.tsx

```diff
@@ -54,24 +67,169 @@ const Template: Story<InfoboxProps> = ({ heading, variant }) => (
+      </Table.Row>
+    </Table.Header>
+    <Table.Body>
+      <Table.Row>
+        <Table.DataCell>Jupiter</Table.DataCell>
+        <Table.DataCell format={"num"}>69,911</Table.DataCell>
+        <Table.DataCell>Gas giant</Table.DataCell>
+        <Table.DataCell format={"date"}>January 21, 2023</Table.DataCell>
+      </Table.Row>
+      <Table.Row>
+        <Table.DataCell>
+          Pluto
+          <Infobox
+            heading={heading}
+            variant={variant}
+            content={content}
+            actions={actions}
+          />
+        </Table.DataCell>
+        <Table.DataCell format={"num"}>6,371</Table.DataCell>
+        <Table.DataCell>Terrestrial</Table.DataCell>
+        <Table.DataCell format={"date"}>January 2, 2021</Table.DataCell>
+      </Table.Row>
+      <Table.Row>
+        <Table.DataCell>Mercury</Table.DataCell>
+        <Table.DataCell format={"num"}>1,737</Table.DataCell>
+        <Table.DataCell>Terrestrial</Table.DataCell>
+        <Table.DataCell format={"date"}>&ndash;</Table.DataCell>
+      </Table.Row>
+    </Table.Body>
+  </Table>
+);
+
 export const Info = Template.bind({});
 Info.args = {
   variant: "info",
+  heading: "Moonbase Alpha-6",
+  content:
+    "You are currently logged in from Moonbase Alpha-6, located on Luna.",
 };
 
 export const Danger = Template.bind({});
 Danger.args = {
   variant: "danger",
+  heading: "Safety checks have failed",
+  content:
+    "An issue has been discovered with your fuel mixture ratios. Please reconfigure your fuel mixture and perform safety checks again.",
 };
 
 export const Caution = Template.bind({});
 Caution.args = {
   variant: "caution",
+  heading: "Safety checks incomplete",
+  content:
+    "Safety checks must be completed before this mission can be approved for launch.",
 };
 
 export const Success = Template.bind({});
 Success.args = {
   variant: "success",
+  heading: "Ready for lift-off",
+  content:
+    "Safety checks are complete, and this mission has been approved for launch.",
 };
 
-export const ContentOnly = (): ReactElement => <Infobox content={content} />;
+export const WithLink = Template.bind({});
+WithLink.args = {
+  variant: "danger",
+  heading: "Safety checks have failed",
+  content:
+    "An issue has been discovered with your fuel mixture ratios. Please reconfigure your fuel mixture and perform safety checks again.",
+  actions: (
+    <>
+      <Link href="#" variant="monochrome">
+        Visit fueling console
+      </Link>
+    </>
+  ),
+};
+
+export const WithLinkInline = Template.bind({});
+WithLinkInline.args = {
+  variant: "danger",
+  heading: "Safety checks have failed",
+  content: (
+    <>
+      An issue has been discovered with your fuel mixture ratios. Please{" "}
+      <Link href="#" variant="monochrome">
+        reconfigure your fuel mixture
+      </Link>{" "}
+      and perform safety checks again.
+    </>
+  ),
+};
+
+export const FormDo = FormTemplate.bind({});
+FormDo.decorators = [
```

Comment:
Alternately, we could utilize `[id$="-do"]` and `[id$="don't"]` selectors as above instead of decorators for Dos and Don'ts. That would remove a small amount of story-writing overhead.

I went with the standard approach here since it's more explicit.

Reply:
we might consider moving the strings for Do/Don't and the icon into the markup here rather than using css pseudo elements and content. We might also consider extracting the decorator itself (liens 211-214) below into a separate module so we can import it where it is needed and share it around more easily!

Reply:
:100: on moving Docs styles into a module/modules. The SB-standard way of managing custom CSS feels v stunted. It'd be awesome if we could hook directly into our tokens too.

Would you mind setting that up next sprint? The appropriate asset pipeline for our setup is still mysterious to me.